tests/unit/test_collector.py

```diff
@@ -195,10 +195,23 @@ def test_get_machine_type(self, collector_daemon, mac_address, expect_machine_ty
 
         assert machine_type.value == expect_machine_type
 
-    def test_get_machine_type_interface_skip(self, collector_daemon):
-        """Test that blacklisted interfaces are not used to detect machine type."""
+    @pytest.mark.parametrize(
+        "skip_interfaces, expected_type",
+        [
+            ([r"^virbr\d*", r"^tap-"], MachineType.METAL),
```

Comment:
interesting, does the `r` prefix not cause problems? The config option should be a plain string afaiu

Reply:
It throws warnings without the `r` complaining about invalid escape sequence. but tests pass either way.